lib/lambda/sinkSubtypes.ts

```diff
@@ -1,6 +1,5 @@
 import { Handler } from "aws-lambda";
-import { KafkaRecord, opensearch } from "shared-types";
-import { KafkaEvent } from "shared-types";
+import { KafkaRecord, opensearch, KafkaEvent } from "shared-types";
```

Comment:
Shouldnt it be KafkaEvent, KafkaRecord, opensearch in order?  Weird the linter didnt pick that up

Reply:
Yes, good point. The order of named imports `{ KafkaRecord, opensearch, KafkaEvent }` follows the export order from their module, so it's not random